test/integration/router/router_certs.go

```diff
@@ -0,0 +1,101 @@
+// +build integration,!no-docker,docker
+package router
```

Comment:
You need to add a space here so this gets interpreted as a build tag, or remove the build tag.

Also, add a comment about how someone regenerates these (the exact config)